scripts/irods/test/test_ifsck.py

```diff
@@ -83,3 +83,31 @@ def test_ifsck_continues_when_user_cannot_access_subdirectory__issue_5358(self):
             for d in dirs:
                 os.chmod(d, 0o777)
 
+    @unittest.skipIf(test.settings.RUN_IN_TOPOLOGY, "Skip for Topology Testing, checks vault")
+    def test_ifsck_unregistered_file_detected__6467(self):
```

Comment:
Consider using more words in the name of the test. For example:
`test_ifsck_unregistered_file_detected__6467` -> `test_ifsck_returns_error_when_an_unregistered_file_is_detected__issue_6467`

Reply:
done